packages/compass-collection/src/components/collection/collection.tsx

```diff
@@ -160,6 +146,7 @@ const Collection: React.FunctionComponent<CollectionProps> = ({
           pipeline={pipeline}
           sourceName={sourceName}
           stats={stats}
+          darkMode={darkMode}
```

Comment:
As this `Collection` component is no longer the user of the `darkMode` prop, it's passed down, it's probably a bit cleaner to remove the property from this component and the `withTheme` wrapper and instead have the `withTheme` wrapper inside of `CollectionHeader.tsx`